AppCenterData/AppCenterData/Internal/Util/MSDocumentUtils.m

```diff
@@ -120,7 +120,8 @@ + (MSDocumentWrapper *)documentWrapperFromDictionary:(NSObject *)object
   if (![MSDocumentUtils isReferenceDictionaryWithKey:object key:kMSDocumentIdKey keyType:[NSString class]] ||
       ![MSDocumentUtils isReferenceDictionaryWithKey:object key:kMSDocumentTimestampKey keyType:[NSNumber class]] ||
       ![MSDocumentUtils isReferenceDictionaryWithKey:object key:kMSDocumentEtagKey keyType:[NSString class]] ||
-      ![MSDocumentUtils isReferenceDictionaryWithKey:object key:kMSPartitionKey keyType:[NSString class]]) {
+      ![MSDocumentUtils isReferenceDictionaryWithKey:object key:kMSPartitionKey keyType:[NSString class]] ||
+      ![MSDocumentUtils isReferenceDictionaryWithKey:object key:kMSDocumentKey keyType:[NSDictionary class]]) {
```

Comment:
That's not correct as delete has null customer payload.

Reply:
And this is what makes unit test fails.

Reply:
Fixed in https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1561/commits/712173afaa8b24c169608caacbc00d922eae9a5a